src/chains/ethereum_1/ethAction.ts

```diff
@@ -85,8 +84,13 @@ export class EthereumActionHelper {
       contract,
     } = actionInput
 
+    const gasPriceInWei =
```

Comment:
Create a helper function and call it
  gweiToWeiString(value: EthereumMultiValue): string {
    // if string .. convert to number
    // if number, toWei()
    // return string
  }

Reply:
Be sure to update set gasPrice to use new helper function